### PR TITLE
Add Jenkinsfile

### PR DESCRIPTION
**Pull request by yma88**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            steps {
+                sh 'printenv'
+            }
+        }
+        stage('Build') {
+            when {
+                expression { env.CHANGE_ID != null } // Pull request
+            }
+            steps {
+                sh '${M2_HOME}/bin/mvn -B -V clean verify sonar:sonar -Prun-its'
```
Comment (ligangty) [inline Jenkinsfile:14]: sonar is not working. Let's remove it. And I don't think this project needs its, so remove it too.

Comment (yma88) [reply Jenkinsfile]: @ligangty Ok, removed.

Comment (ligangty) [reply Jenkinsfile]: I mean remove -Prun-its, but not add -Pci. They are both unavailable.

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            steps {
+                sh 'printenv'
+            }
+        }
+        stage('Build') {
+            when {
+                expression { env.CHANGE_ID != null } // Pull request
+            }
+            steps {
+                sh '${M2_HOME}/bin/mvn -B -V clean verify -Prun-its'
```
Comment (ligangty) [inline Jenkinsfile:14]: I mean this -Prun-its is also not needed.

**Review by ligangty — COMMENTED** [review]
LGTM

**Review by ligangty — APPROVED** [review]
LGTM

**Pull request merged**